standalone/ansible/group_vars/all.yml

```diff
@@ -2,7 +2,7 @@
 ansible_port: 22
 deploy_user: "deploy"
 ruby_version: "2.6.6"
-bundler_version: "1.17.3"
+bundler_version: "2.2.29"
```

Comment:
These vars will be duplicated between standalone and non-standalone, but at least now we have vars in a consistent spot and can easily make changes for upgrades across all deployments.

I'm not sure how we would tackle resolving common vars across all deployments, and don't think its worth the effort to take that on right now.